index.html

```diff
@@ -0,0 +1,77 @@
+                <img src="images/bg-card-back.png" alt="">
+                <div class = "cardFront">
+                    <div>
+                        <img src="images/card-logo.svg" alt="">
+                    </div>
+                    <div class = "cardNumberDisplay">0000000000000000</div>
+                    <div>
+                        <span class = "cardNameDisplay">Ojo Gbotemi</span>
+                        <span>
+                            <span class = "displayMonth">00</span>
+                            <span>/</span>
+                            <span class = displayYear>00</span>
+                        </span>
+                    </div>
+                </div>
+                <div class = "cardBack">
+                    <div class="displayCVC">000</div>
+                </div>
+            </div>
+        </div>
+        <div class = "formContainer">
+            <div class = "theForm">
+                <form>
+                    <span class = "firstSpan">
+                        <label for="CardHolder">CARDHOLDER NAME</label>
+                        <input type="text" name="Card-Holder" id="cardHolder" placeholder="e.g. Ojo Gbotemi" minLength="6" maxLength ="30" required>
+                        <div class="incorrectName"></div>
+                    </span>
+                    <span>
+                        <label for="cardNumber">CARD NUMBER</label>
+                        <input type="number" name="card-Number" id="cardNumber" placeholder="e.g. 1234 5678 9123 0000" minLength="16" maxlength="16" required>
+                        <div class="incorrectNumber"></div>
+                    </span>
+                    <div class = "bottomContainer">
+                        <div>
+                            <span>EXP.DATE[MM/YY]</span>
+                            <input type="number" name="" id="month" placeholder="MM" maxLength="2" min=1 max=12 minLength="2" required>
+                            <input type="number" name="" id="year" placeholder="YY" maxLength="2" required minLength="2">
+                            <div class="incorrectDate"></div>
+                        </div>
+                        <div>
+                            <label for="cvc">CVC</label>
+                            <input type="number" name="CVC" id="cvc" placeholder="e.g. 123" maxLength="3" required minLength="3">
+                            <div class = "incorrectCVC"></div>
+                        </div>
+                    </div>
+                    <button type="reset" class = "submitBtn">Confirm</button>
+                </form>
+            </div>
+        </div>
+        <div class = "completeState">
+            <img src="images/icon-complete.svg" alt="">
+            <div>THANK YOU!</div>
+            <div>We've added your card details</div>
+            <button class = "Continue">Continue</button>
+        </div>
+    </div>
+    <script src="script.js"></script>
+</body>
+</html>
```

Comment:
Add a new line at the end of this file. If you are curious about why: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline